website/blog/2025-12-16-aks-config-scheduler-profiles-preview/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Announcing Azure Configurable Scheduler Profiles: Optimize resource ROI with fine grained scheduling strategies."
+description: "Avoid scheduling inefficiencies and improve GPU utilization with AKS Configurable Scheduler Profiles"
+date: 2025-12-16
+authors: 
+   - Colin Mixon
+categories: 
+- operations
+- ai
+tags:
+  - AI
+  - performance
+  - Scheduler
+---
+# Avoid scheduling inefficiencies and improve GPU utilizaiton with AKS Configurable Scheduler Profiles (Preview)
+
+Thoughtful scheduling strategies can resolve pervasive challenges across web-distributed workloads and AI workloads like resiliency and resource utilization. But the default scheduler was primarily designed for general-purpose workloads and out-of-box pod scheduling. The scheduler ultimately selects the optimal node for pod(s) based on several criteria, including (but not limited to):
+
+1. Resource requirements (CPU, memory)
+2. Node affinity/anti-affinity
+3. Pod affinity/anti-affinity
+4. Taints and tolerations
+
+The criteria, and their respective priority in the scheduling cycle, are not suitable for advanced use cases that might require custom scheduling strategies. Nor, does the default scheduler enable user customization for fine-grain pod placement control while avoiding managing a second custom scheduler. For example, users running batch jobs might prefer collocating on a few nodes for better performance or cost-sensitive workloads might benefit from node binpacking to minimize idle node costs.
+
+To support these advanced use cases, and to give users more control, use [AKS Configurable Scheduler Profiles][concepts-scheduler-configuration] to tailor a scheduler to their specific workload requirements using node bin-packing, preemption, and 16 other scheduling plugins that can optimize ROI​, improve gpu utilization, improve data locality, or increase resliency.
+
```

Comment:
You need to make the problem clear. We are not inventing all those plugins. There is a configuration problem in AKS and we try to resolve the configuration problem.